Somesh/HTML/Assignment6/que1Assign6.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Question1Assignment6</title>
+        <style>
+            h1,
+            h2 {
+                background-color: rgb(0, 163, 109);
+                text-align: center;
+                color: white;
+                font-family: Poppins;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>MI-11 Intern Html Assignent</h1>
+        <br />
```

Comment:
multiple  time br tag used in that place manage with margin

Reply:
Not a problem we can use this as well.

Reply:
but this code not follow coding standerd